.github/workflows/deploy.yml

```diff
@@ -224,6 +224,44 @@ jobs:
                   function_name: sendSessionInsightsEmails
                   zip_file: backend/sendSessionInsightsEmails.zip
 
+    deploy-user-journey-lambdas:
```

Comment:
i'll file a ticket for the next tech debt week to see if we can clean up this file to avoid having to copy-paste for each lambda